src/noshow/api/app_helpers.py

```diff
@@ -148,6 +154,13 @@ def create_treatment_groups(
         )
     else:
         predictions.loc[:, "treatment_group"] = None
+
+    # Treatment group 2 means excluded from RCT
+    if rct_agendas:
+        predictions.loc[
```

Comment:
maybe nice to also add this to the unit test as it is quite important that this always works

Reply:
Good point, added two unit tests